### DISCUSSION TITLE
Report from Log files

### DISCUSSION THREAD

**Discussion by Ekat1** [ThioJoe/YT-Spammer-Purge#608] [Ideas]
Hi, I noticed that when trying to go through scanned comments from a log file after my scan crashed, the log scan option only allowed hiding or deleting comments, which you cannot do if you don't own the video. My idea is to add an option to report from log files.

**Comment by Firecul** [comment]
Please see #569

**Comment by ThioJoe** [comment]
locking as duplicate
